content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,707 @@
+      directly on a friend's timeline. Click the :guilabel:`Share to News Feed or Story` drop-down
+      menu near the top of the page, and select :guilabel:`Share on a Fried's Timeline`.
+
+      When selected, a :guilabel:`Friend` field appears below the selection. Begin to type in the
+      friend's name. As the name is typed, a list appears, populated with names that match what is
+      entered. Select the friend from the list. Only one friend can be selected.
+
+      When the post is ready to be shared, click the blue :guilabel:`Post to Facebook` button at the
+      bottom of the screen.
+
+   .. tab:: Share in a group
+
+      To reach a specific audience and not share the job position publicly with everyone, a job can
+      be shared in a group. Click the :guilabel:`Share to News Feed or Story` drop-down menu near
+      the top of the page, and select :guilabel:`Share in a Group`.
+
+      When selected, a :guilabel:`Group` field appears below the selection. The available groups
+      that can be posted to are groups the user is currently a member of. Job positions cannot be
+      shared with groups the user has not joined. Begin to type in the name of the group. As the
+      group name is typed, a list appears, populated with the group names that match what is
+      entered. Select the group from the list. Only one group can be selected.
+
+      When the post is ready to be shared, click the blue :guilabel:`Post to Facebook` button at the
+      bottom of the screen.
+
+   .. tab:: Share in an event
+
+      A job position can also be shared on an event page. Click the :guilabel:`Share to News Feed or
+      Story` drop-down menu near the top of the page, and select :guilabel:`Share in an Event`.
+
+      When selected, a :guilabel:`Event` field appears below the selection. The available events
+      that can be posted to are events the user is invited to. This includes events that the user
+      has not responded to yet, and events that have occurred in the past. Begin to type in the name
+      of the event. As the event name is typed, a list appears, populated with the events that match
+      what is entered. Select the event from the list. Only one event can be selected.
+
+      When the post is ready to be shared, click the blue :guilabel:`Post to Facebook` button at the
+      bottom of the screen.
+
+   .. tab:: Share in a private message
+
+      The last option is to share a job position privately instead of publicly, in a Facebook
+      message. Click the :guilabel:`Share to News Feed or Story` drop-down menu near the top of the
+      page, and select :guilabel:`Share in an Private Message`.
+
+      When selected, a :guilabel:`To` field appears below the selection. Begin to type in the
+      friend's name. As the name is typed, a list appears, populated with names that match what is
+      entered. Select the friend from the list. Only one friend can be selected.
```

Comment:
!!!!!!  What?!  I tested this and thought it only allowed one. I now realize, since I typed in random letters- I must have typed 2 or 3 random letters that simply didn't match any of my friend's names! Thanks for this catch.